### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by domesticmouse**
## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: veggieseasons/lib/widgets/trivia.dart [hunk 1]
```diff
@@ -40,8 +40,8 @@ class _TriviaViewState extends State<TriviaView> with RestorationMixin {
   Trivia get currentTrivia => veggie.trivia[triviaIndex.value];
 
   /// The current state of the game.
-  _RestorablePlayerStatus status =
-      _RestorablePlayerStatus(PlayerStatus.readyToAnswer);
+  RestorablePlayerStatus status =
```
Comment (domesticmouse) [inline veggieseasons/lib/widgets/trivia.dart:43]: revert?

File: veggieseasons/lib/widgets/trivia.dart [hunk 2]
```diff
@@ -222,8 +222,8 @@ class _TriviaViewState extends State<TriviaView> with RestorationMixin {
   }
 }
 
-class _RestorablePlayerStatus extends RestorableValue<PlayerStatus> {
-  _RestorablePlayerStatus(this._defaultValue);
+class RestorablePlayerStatus extends RestorableValue<PlayerStatus> {
```
Comment (domesticmouse) [inline veggieseasons/lib/widgets/trivia.dart:225]: revert?

Comment (craiglabenz) [reply veggieseasons/lib/widgets/trivia.dart]: Done

File: add_to_app/plugin/flutter_module_using_plugin/lib/cell.dart [hunk 3]
```diff
@@ -113,13 +113,16 @@ class _CellState extends State<Cell> with WidgetsBindingObserver {
                             : Stream.value(defaultPosition),
                         initialData: defaultPosition,
                         builder: (context, snapshot) {
+                          if (snapshot.data == null) {
```
[the remark this answers is not in the document]
Comment (craiglabenz) [reply add_to_app/plugin/flutter_module_using_plugin/lib/cell.dart]: Done

File: add_to_app/multiple_flutters/multiple_flutters_module/lib/main.dart [hunk 4]
```diff
@@ -36,10 +36,10 @@ class MyHomePage extends StatefulWidget {
   final String title;
 
   @override
-  _MyHomePageState createState() => _MyHomePageState();
+  MyHomePageState createState() => MyHomePageState();
```
[the remark this answers is not in the document]
Comment (craiglabenz) [reply add_to_app/multiple_flutters/multiple_flutters_module/lib/main.dart]: Done

File: animations/lib/src/misc/card_swipe.dart [hunk 5]
```diff
@@ -10,10 +10,10 @@ class CardSwipeDemo extends StatefulWidget {
   static String routeName = '/misc/card_swipe';
 
   @override
-  _CardSwipeDemoState createState() => _CardSwipeDemoState();
+  CardSwipeDemoState createState() => CardSwipeDemoState();
```
[the remark this answers is not in the document]
Comment (domesticmouse) [reply animations/lib/src/misc/card_swipe.dart]: Hmm, this appears out of date. Weird.

**Comment by domesticmouse** [comment]
PTAL @craiglabenz

**Review by craiglabenz — APPROVED** [review]
LGTM

**Pull request merged**